### PR TITLE
refactor: refactor imports PE-1365

### PR DESCRIPTION
**Pull request by jdaev**
Also removes deprecated tests

### CODE REVIEW

**Review by karlprieb — DISMISSED** [review]

File: lib/components/app_drawer/app_drawer.dart [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 import 'package:ardrive/misc/resources.dart';
 import 'package:ardrive/theme/theme.dart';
+import 'package:ardrive/utils/app_localizations_wrapper.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:package_info_plus/package_info_plus.dart';
 import 'package:url_launcher/url_launcher.dart';
 
-import '../../../utils/app_localizations_wrapper.dart';
 import '../components.dart';
```
Comment (thiagocarvalhodev) [inline lib/components/app_drawer/app_drawer.dart:10]: import 'components.dart';

File: lib/pages/app_router_delegate.dart [hunk 2]
```diff
@@ -5,11 +5,11 @@ import 'package:ardrive/entities/constants.dart';
 import 'package:ardrive/models/models.dart';
 import 'package:ardrive/pages/pages.dart';
 import 'package:ardrive/services/services.dart';
+import 'package:ardrive/utils/app_localizations_wrapper.dart';
 import 'package:cryptography/cryptography.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 
-import '../../../utils/app_localizations_wrapper.dart';
 import '../app_shell.dart';
```
Comment (thiagocarvalhodev) [inline lib/pages/app_router_delegate.dart:13]: This one is relative too

File: lib/pages/drive_detail/components/drive_file_drop_zone.dart [hunk 3]
```diff
@@ -2,13 +2,13 @@ import 'package:ardrive/blocs/blocs.dart';
...
 import 'package:ardrive/services/services.dart';
+import 'package:ardrive/utils/app_localizations_wrapper.dart';
 import 'package:ardrive/utils/upload_plan_utils.dart';
 import 'package:file_selector/file_selector.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:flutter_dropzone/flutter_dropzone.dart';
 
-import '../../../utils/app_localizations_wrapper.dart';
 import '../../congestion_warning_wrapper.dart';
```
Comment (thiagocarvalhodev) [inline lib/pages/drive_detail/components/drive_file_drop_zone.dart:12]:
```suggestion
import 'package:ardrive/pages/congestion_warning_wrapper.dart'
```

**Review by thiagocarvalhodev — APPROVED** [review]

File: lib/components/app_drawer/app_drawer.dart (hunk 1 above)
Comment (thiagocarvalhodev) [reply lib/components/app_drawer/app_drawer.dart:10]:
```suggestion
import 'components.dart';
```

File: lib/pages/app_router_delegate.dart (hunk 2 above)
Comment (thiagocarvalhodev) [inline lib/pages/app_router_delegate.dart:13]:
```suggestion
import 'package:ardrive/app_shell.dart';
```

**Review by thiagocarvalhodev — APPROVED** [review]

File: lib/pages/app_router_delegate.dart (hunk 2 above)
Comment (jdaev) [reply lib/pages/app_router_delegate.dart]: [Updated](https://github.com/ardriveapp/ardrive-web/pull/461/commits/dd6b0f30afb704b18dcabffebf06a8a7a985b990)

File: lib/pages/drive_detail/components/drive_file_drop_zone.dart (hunk 3 above)
Comment (jdaev) [reply lib/pages/drive_detail/components/drive_file_drop_zone.dart]: [Updated](https://github.com/ardriveapp/ardrive-web/pull/461/commits/dd6b0f30afb704b18dcabffebf06a8a7a985b990)

File: lib/components/app_drawer/app_drawer.dart (hunk 1 above)
Comment (jdaev) [reply lib/components/app_drawer/app_drawer.dart]: [Updated](https://github.com/ardriveapp/ardrive-web/pull/461/commits/dd6b0f30afb704b18dcabffebf06a8a7a985b990)

**Review by thiagocarvalhodev — APPROVED** [review]

**Pull request merged**